### DISCUSSION TITLE
Add Striver's SDE sheet

### DISCUSSION THREAD

**Discussion by av1choudharyy** [AkashSingh3031/The-Complete-FAANG-Preparation#102] [Ideas]
Add Striver's SDE sheet. An easy 30 days roadmap.

**Comment by AkashSingh3031** [comment]
> Add Striver's SDE sheet. An easy 30 days roadmap.

Ok, I will add this
Thanks for suggestions

**Reply by AkashSingh3031** [reply]
@av1choudharyy 
Added **Striver 30 Days of SDE Sheet**

If any new suggestion, then feel free to create an issue
